cmd/cstor-pool-mgmt/controller/common/common.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Why is this file named as common.go? Is this some specified standard?

Reply:
Since we are keeping some part of code common to cstorpool and cvr controller, we are naming it common. Plz suggest if you want a different name.

Reply:
As discussed, we can use the name common.